docs/docs/user-setup.md

```diff
@@ -199,7 +199,7 @@ storage device. Once the flash is complete, the controller should automatically
 
 ZMK will automatically advertise itself as connectable if it is not currently connected to a device. You should be able to see your keyboard from the bluetooth scanning view of your laptop or phone / tablet. It is reported by some users that the connections with Android / iOS devices are generally smoother than with laptops, so if you have trouble connecting, you could try to connect from your phone or tablet first to eliminate any potential hardware issues.
 
-ZMK support multiple BLE “profiles”, which allows you to connect to and switch among multiple devices. Please refer to the [Bluetooth behavior](behaviors/bluetooth.md) section for detailed explanations of how to use them.
+ZMK supports multiple BLE “profiles”, which allows you to connect to and switch amongst multiple devices. Please refer to the [Bluetooth behavior](behaviors/bluetooth.md) section for detailed explanations on how to use them.
```

Comment:
Looks good to me, just a comment that "amongst" seems to be equivalent to "among" but less commonly used in American English, so I'd prefer the former.

Reply:
Thanks! So you would prefer "amongst"?

Reply:
Yeah sorry, typo there. I'd prefer "among" :)

Reply:
All good, changed it back haha